languageserver/protocol/types.go

```diff
@@ -2816,7 +2816,7 @@ type CompletionItem struct {
 	/*Documentation defined:
 	 * A human-readable string that represents a doc-comment.
 	 */
-	Documentation string `json:"documentation,omitempty"` // string | MarkupContent
+	Documentation interface{} `json:"documentation,omitempty"` // string | MarkupContent
```

Comment:
Why's this an interface? The comments say string 🤔

Reply:
It can be either a string or [`MarkupContent`](https://microsoft.github.io/language-server-protocol/specification#markupContent). In Go this can only be represented as `interface{}` (or adding a new type)